### ISSUE TITLE
Support/handle negative replication lag

### ISSUE THREAD

**Issue by shlomi-noach** [openark/orchestrator#1072]
The current database schema defines:
```
          seconds_behind_master bigint(20) unsigned DEFAULT NULL,
          slave_lag_seconds bigint(20) unsigned DEFAULT NULL,
```

`unsugned` _makes sense_, until it doesn't. Clock skew might lead to (false) negative replication lag. With `unsigned`, `orchestrator` uses the absolute value of that negative lag.

In a recent production issue reported and analyzed by @ggunson, a clock was skewed by `744s` ; the replica was _ahead_ of the master in terms of internal clock. `orchestrator` reported the replica as lagging by `744s`.

We should either support negative values in the backend database (this will be a challenge to fix on `sqlite` backends), or have `orchestrator` identify the scenario before persisting to database and make some decision (e.g. write `0` to database, indicate some error).

`orchestrator` can possibly read the system time on each server and report skews as a Structure Warning.

**Comment by sjmudd** [comment]
I agree with the assessment that the negative delay should not ever happen, but given the way the value is/could be collected it's quite possible for the collected value to be negative.
* being able to collect such values means it's easier to handle it. Using the absolute value instead is only confusing the issue.
* such negative values should be treated as errors
